src/gui/gui-hotlist.c

```diff
@@ -474,6 +474,86 @@ gui_hotlist_resort ()
     gui_hotlist_changed_signal ();
 }
 
+struct t_gui_hotlist *
+gui_hotlist_get ()
```

Comment:
This looks like a verbose copy of `gui_hotlist_find_pos`. Maybe one function could be performing the same logic without the need for duplicate code, especially considering that there could be just one enum.

Reply:
Correct, it is a copy of `gui_hotlist_find_pos`.
I'll try to pick the common part out and reorganize the code.